x-pack/metricbeat/module/stan/_meta/kibana/7/dashboard/Metricbeat-STAN-overview.json

```diff
@@ -127,15 +110,14 @@
                   {
                     "field": "stan.channels.messages",
                     "id": "61ca57f2-469d-11e7-af02-69e470af7417",
-                    "type": "sum"
+                    "type": "max"
                   }
                 ],
                 "point_size": 1,
                 "separate_axis": 0,
-                "split_color_mode": "gradient",
                 "split_mode": "terms",
                 "stacked": "none",
-                "terms_field": "stan.channels.name.keyword",
+                "terms_field": "stan.channels.name",
```

Comment:
Should channel names not be a keyword based query? I had difficulty when this wasn't a keyword